File: fix_store.py
import re

with open('src/stores/adminBusinessConsultingStore.js', 'r', encoding='utf-8') as f:
    c = f.read()

c = c.replace('useConsultingDossierStore', 'useAdminBusinessConsultingStore')
c = c.replace(\"'consultingDossier'\", \"'adminBusinessConsulting'\")
c = re.sub(r\"doc\\(db, 'programs', programId, 'consultingDossiers', dossierId\\)\", \"doc(db, 'businesses', businessId, 'consulting', 'consulting')\", c)
c = c.replace(\"loadProgramDossiers(programId)\", \"loadProgramDossiers(businessId)\")
c = c.replace(\"loadDossier(programId, dossierId)\", \"loadDossier(businessId)\")
c = c.replace(\"updateDossierStep(programId, dossierId, nextStep)\", \"updateDossierStep(businessId, nextStep)\")
c = c.replace(\"saveS0Assessment(programId, dossierId, payload)\", \"saveS0Assessment(businessId, payload)\")
c = c.replace(\"saveProgramConsultingDossier(programId, dossierId, payload)\", \"saveProgramConsultingDossier(businessId, payload)\")
c = c.replace(\"markCycleCompleted(programId, dossierId, cycleKey)\", \"markCycleCompleted(businessId, cycleKey)\")
c = c.replace(\"reopenCycleWithCascade(programId, dossierId, cycleKey)\", \"reopenCycleWithCascade(businessId, cycleKey)\")
c = c.replace(\"createDossier(programId, participant)\", \"createDossier(businessId, participant)\")

with open('src/stores/adminBusinessConsultingStore.js', 'w', encoding='utf-8') as f:
    f.write(c)

File: fix_store.js
const fs = require('fs');

let c = fs.readFileSync('src/stores/consultingDossierStore.js', 'utf8');
c = c.replace(/useConsultingDossierStore/g, 'useAdminBusinessConsultingStore');
c = c.replace(/'consultingDossier'/g, \"'adminBusinessConsulting'\");
c = c.replace(/loadProgramDossiers\(programId\)/g, \"loadProgramDossiers(businessId)\");
c = c.replace(/loadDossier\(programId, dossierId\)/g, \"loadDossier(businessId)\");
c = c.replace(/updateDossierStep\(programId, dossierId, nextStep\)/g, \"updateDossierStep(businessId, nextStep)\");
c = c.replace(/saveS0Assessment\(programId, dossierId, payload\)/g, \"saveS0Assessment(businessId, payload)\");
c = c.replace(/saveProgramConsultingDossier\(programId, dossierId, payload\)/g, \"saveProgramConsultingDossier(businessId, payload)\");
c = c.replace(/markCycleCompleted\(programId, dossierId, cycleKey\)/g, \"markCycleCompleted(businessId, cycleKey)\");
c = c.replace(/reopenCycleWithCascade\(programId, dossierId, cycleKey\)/g, \"reopenCycleWithCascade(businessId, cycleKey)\");
c = c.replace(/createDossier\(programId, participant\)/g, \"createDossier(businessId, participant)\");

// Inside loadProgramDossiers
c = c.replace(/collection\(db, 'programs', programId, 'consultingDossiers'\)/g, \"collection(db, 'businesses', businessId, 'consulting')\");

// Inside loadDossier, updateDossierStep, saveS0Assessment, saveProgramConsultingDossier, markCycleCompleted, reopenCycleWithCascade
c = c.replace(/doc\(db, 'programs', programId, 'consultingDossiers', dossierId\)/g, \"doc(db, 'businesses', businessId, 'consulting', 'consulting')\");

// replace "programs", programId, "consultingDossiers", dossierId references
c = c.replace(/dossierId/g, \"businessId\");
c = c.replace(/programId/g, \"businessId\");

fs.writeFileSync('src/stores/adminBusinessConsultingStore.js', c);
